buildconfig/Jenkins/buildscript

```diff
@@ -22,8 +23,20 @@ if [[ ${NODE_LABELS} == *ubuntu* ]]; then
 fi
 if [[ ${NODE_LABELS} == *osx* ]]; then
   ON_MACOS=true
+  USE_CORE_DUMPS=false
 fi
 
+###############################################################################
+# Script cleanup
+###############################################################################
+ULIMIT_CORE_ORIG=$(ulimit -c)
+function onexit {
+  if [[ ${USE_CORE_DUMPS} == true ]]; then
+    ulimit -c $ULIMIT_CORE_ORIG
+  fi
+}
+trap onexit EXIT
```

Comment:
According to https://unix.stackexchange.com/questions/57940/trap-int-term-exit-really-necessary `EXIT` is interpreted differently by different shells, do we need to specify this differently?

Reply:
Thanks for pointing this out. I hadn't realised that shells interpreted this differently. The Jenkins plugin requires the script to be executable and we have the shebang line `/bin/bash -ex` to force `bash` in this case.
Perhaps we should add `SIGINT` & `SIGTERM` too just in case plugin behaviour changes?

Reply:
Sound sufficient. :+1: